web/api/v1/api.go

```diff
@@ -1317,8 +1317,8 @@ func (api *API) serveFlags(_ *http.Request) apiFuncResult {
 	return apiFuncResult{api.flagsMap, nil, nil, nil}
 }
 
-// stat holds the information about individual cardinality.
-type stat struct {
+// TSDBStat holds the information about individual cardinality.
+type TSDBStat struct {
```

Comment:
I feel `TSDBStat` is too general. If it is about cardinality maybe we can call it `CardinalityStat` or sth else?

Reply:
I wouldn't mind changing the name to `CardinalityStat`, but it's also worth noting that these stats also expose `MemoryInBytesByLabelName`. So I am not sure if that is considered cardinality. 

It's also possible that the tsdb status could contain other fields in the future that are not strictly related to cardinality. Since this struct will be exported now, it might be better to keep it more generic and avoid renaming it in the future. Wdyt?